pkg/repo/repotest/server.go

```diff
@@ -248,21 +263,21 @@ func (srv *OCIServer) Run(t *testing.T, opts ...OCIServerOpt) {
 // the server.
 //
 // Deprecated: use NewTempServerWithCleanup
-func NewTempServer(glob string) (*Server, error) {
+func NewTempServer(glob string) (*Server, string, error) {
```

Comment:
This has an API breaking change.

My quick scan for these issues: I look for functions that start with a capital letter that have changed. If it has, that's breaking.

Reply:
Okay, I was under the impression that anything inside repotest is just used for unit testing. I have updated this.